lib/web_ui/lib/src/engine/mouse/cursor.dart

```diff
@@ -61,15 +51,12 @@ class MouseCursor {
     'zoomIn': 'zoom-in',
     'zoomOut': 'zoom-out',
   };
+
   static String _mapKindToCssValue(String? kind) {
     return _kindToCssValueMap[kind] ?? 'default';
   }
 
   void activateSystemCursor(String? kind) {
-    setElementStyle(
-      flutterViewEmbedder.flutterViewElement,
-      'cursor',
-      _mapKindToCssValue(kind),
-    );
+    element.style.cursor = _mapKindToCssValue(kind);
```

Comment:
The only problem I can imagine with this is that if the `element` gets detached/deleted from the DOM, and then a system cursor event message comes, this might attempt to operate on a null instance. Since this is speculation and not a bug that we've seen so far, I say we don't worry about it for now.

Reply:
That's not a problem at all! The element is non-nullable and cannot become null. Yes, it can be detached/removed from the DOM tree, but it is still a valid element. In fact, in `cursor_test.dart`, we are using detached elements to test `MouseCursor`.